### PR TITLE
`lib/Fn.js` Clean up and Commenting

### PR DESCRIPTION
**Pull request by lyzadanger**
As the first part of creating some docs for `lib/Fn.js` #1214 , I've added and updated commenting in source. I've also shored up tests, though it was in a pretty good place to start with. At this point, we've got fully 100% coverage.

A few notes:

* I didn't try to meaningfully comment the `fma` function because it's, ah, a bit beyond my grade level.
* I was a tad confused by the behavior of the auto-generated `Fn.u*` functions, which I believe are meant to constrain the passed value to a certain bit size. `Fn.u8(256)` returns `256`. Isn't that a 9-bit number? I may completely not understand what those functions are for, though :).

### CODE REVIEW

**Comment by lyzadanger** [comment]
I'm not sure why CI failed... anyway, I've got a WIP Wiki page for docs going now, too: https://github.com/rwaldron/johnny-five/wiki/Fn

I also noticed something:

```
test.equal(Fn.uint32(200, 255, 255, 255), -922746881);
```

This test passes despite the function claiming to generate unsigned ints.

File: lib/fn.js [hunk 1]
```diff
@@ -3,77 +3,106 @@ var Fn = {
...
+ * decimal point.
+ *
+ * @param {Number} number - The number to format
+ * @param {Number} [digits = 0] - The number of digits after the decimal point
+ * @return {Number} Formatted number
+ * @example
+ * Fn.toFixed(5.4564, 2); // -> 5.46
+ * @example
+ * Fn.toFixed(1.5, 2); // -> 1.5
+ */
```
Comment (rwaldron) [inline lib/fn.js]: 👍

**Comment by lyzadanger** [comment]
@rwaldron OK, I'd love another set of eyes on the `Fn.uint32` function and the auto-generated `Fn.u*` functions. I've tried to be careful in these bug fixes but having someone check my work would be nifty. Added more tests and commenting; everything is passing.

And I know yet more about signed and unsigned ints...

**Comment by rwaldron** [comment]
LGTM, merge at will

File: lib/fn.js [hunk 2]
```diff
@@ -97,41 +126,58 @@ Fn.range = function(lower, upper, tick) {
...
+/**
+ * Square your x!
+ *
+ * @param {Number} x
+ * @return {Number| Nan} - x^2—unless you were goofy enough to provide a
+ *   non-numeric x, in which case it's NaN for you!
+ */
 Fn.square = function(x) {
   return x * x;
 };
```
Comment (rwaldron) [inline lib/fn.js]: Turns out this is n't used anywhere—maybe we can removed it?

Comment (lyzadanger) [inline lib/fn.js]: Oh, whoops! Missed this comment. I do think this could safely go :)

**Pull request merged**